about.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet" type="text/css"/>
+
+<title>About</title>
+
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+<link href="style.css" rel="stylesheet"/>
+</head>
+<body>
+    <header class="header">
+        <nav class="first-navigation">
+            <ul class="first-ul">
+                <li><a href="#"><i class="fa fa-facebook social" id="social"></i></a></li>
+                <li><a href="#"><i class="fa fa-twitter" ></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log out</a></li>
+            </ul>
+        </nav>
+        <nav class="sec-nav">
+            <span class="menu" id="menu"></span>  
+                <img class="logo" src="img/MyLogo.png" alt="logo">
+                    <ul class="sec-ul"  id="mobile-menu">
+                        <li><i class="fas fa-times close-btn" id="close-btn" ></i></li>
```

Comment:
- [ ] Close (X) button doesn't appear in about page. Please fix this.

![image](https://user-images.githubusercontent.com/38635158/124987840-f56f2800-e045-11eb-9109-a052827c0314.png)